Packs/Jira/ReleaseNotes/1_4_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- %%UPDATE_RN%%
```

Comment:
Please describe the change here, you can find more information [here](https://xsoar.pan.dev/docs/documentation/release-notes)